src/NuGet.Clients/NuGet.Credentials/PluginCredentialProvider.cs

```diff
@@ -164,14 +242,24 @@ public PluginCredentialProvider(string path, int timeoutSeconds)
                 throw PluginException.CreateNotStartedMessage(Path);
             }
 
-            process.OutputDataReceived += (object o, DataReceivedEventArgs e) => { stdOut.AppendLine(e.Data); };
-            process.ErrorDataReceived += (object o, DataReceivedEventArgs e) => { stdError.AppendLine(e.Data); }; 
+            process.OutputDataReceived += (object o, DataReceivedEventArgs e) => { outBuffer.AppendLine(e.Data); };
+
+            // Trace and error information may be written to standard error by the provider.
+            // It should be logged at the Information level so it will appear if Verbosity >= Normal.
+            process.ErrorDataReceived += (object o, DataReceivedEventArgs e) =>
+            {
+                if (!string.IsNullOrWhiteSpace(e?.Data))
```

Comment:
Looks like we are no longer collecting stderr for the exception message. Intentional?